politeiawww/api/v1/api.md

```diff
@@ -1143,378 +799,44 @@ Reply:
+| ErrorStatusInvalidEmailOrPassword | 1 | Either the user name or password was invalid. |
+| ErrorStatusMalformedEmail | 2 | The provided email address was malformed. |
+| ErrorStatusVerificationTokenInvalid | 3 | The provided user activation token is invalid. |
+| ErrorStatusVerificationTokenExpired | 4 | The provided user activation token is expired. |
+| ErrorStatusProposalMissingFiles | 5 | The provided proposal does not have files. This error may include additional context: index file is missing - "index.md". |
+| ErrorStatusProposalNotFound | 6 | The requested proposal does not exist. |
+| ErrorStatusProposalDuplicateFilenames | 7 | The provided proposal has duplicate files. This error is provided with additional context: the duplicate name(s). |
+| ErrorStatusProposalInvalidTitle | 8 | The provided proposal title is invalid. This error is provided with additional context: the regular expression accepted. |
+| ErrorStatusMaxMDsExceededPolicy | 9 | The submitted proposal has too many markdown files. Limits can be obtained by issuing the Policy command. |
+| ErrorStatusMaxImagesExceededPolicy | 10 | The submitted proposal has too many images. Limits can be obtained by issuing the Policy command. |
+| ErrorStatusMaxMDSizeExceededPolicy | 11 | The submitted proposal markdown is too large. Limits can be obtained by issuing the Policy command. |
+| ErrorStatusMaxImageSizeExceededPolicy | 12 | The submitted proposal has one or more images that are too large. Limits can be obtained by issuing the Policy command. |
+| ErrorStatusMalformedPassword | 13 | The provided password was malformed. |
+| ErrorStatusCommentNotFound | 14 | The requested comment does not exist. |
+| ErrorStatusInvalidProposalName | 15 | The proposal's name was invalid. |
+| ErrorStatusInvalidFileDigest | 16 | The digest (SHA-256 checksum) provided for one of the proposal files was incorrect. This error is provided with additional context: The name of the file with the invalid digest. |
+| ErrorStatusInvalidBase64 | 17 | The name of the file with the invalid encoding.The Base64 encoding provided for one of the proposal files was incorrect. This error is provided with additional context: the name of the file with the invalid encoding. |
+| ErrorStatusInvalidMIMEType | 18 | The MIME type provided for one of the proposal files was not the same as the one derived from the file's content. This error is provided with additional context: The name of the file with the invalid MIME type and the MIME type detected for the file's content. |
+| ErrorStatusUnsupportedMIMEType | 19 | The MIME type provided for one of the proposal files is not supported. This error is provided with additional context: The name of the file with the unsupported MIME type and the MIME type that is unsupported. |
+| ErrorStatusInvalidPropStatusTransition | 20 | The provided proposal cannot be changed to the given status. |
 
 ### Proposal status codes
```

Comment:
Same comment as the error codes - the links to proposal status codes throughout the doc seem to be broken.